src/containers/deactivate-account-page/DeactivateAccountPage.module.css

```diff
@@ -0,0 +1,97 @@
+.tile {
+  flex: 1;
+  border-radius: 6px;
+  box-shadow: 0px 16px 20px var(--table-shadow);
+  width: 100%;
+  background-color: var(--white);
+  text-align: left;
+  padding: 50px;
+}
+
+.header {
+  font-weight: var(--font-bold);
+  font-size: var(--font-2xl);
+  color: var(--secondary);
+  margin: 10px 0;
+}
+
+.list {
+  list-style-type: disc;
+  line-height: 32px;
+  margin-inline-start: 1.5em;
+  margin-bottom: 50px;
+  font-weight: var(--font-demi-bold);
+}
+.list li:last-child {
```

Comment:
i like this grouping, but i think @sliptype tends to not -- we should agree on a convention / find a good css linter

Reply:
@sliptype   🤜  FIGHT 🤛  FIGHT 👊  FIGHT

Reply:
I have no dog in this fight